indico/modules/events/registration/blueprint.py

```diff
@@ -59,6 +64,17 @@
                  'schedule_regform', RHRegistrationFormSchedule, methods=('GET', 'POST'))
 _bp.add_url_rule('/manage/registration/<int:reg_form_id>/form/', 'modify_regform', RHRegistrationFormModify)
 
+# Registrations management
+_bp.add_url_rule('/manage/registration/<int:reg_form_id>/registrations/', 'manage_reglist', RHRegistrationsListManage)
+_bp.add_url_rule('/manage/registration/<int:reg_form_id>/registrations/customize', 'customize_reglist',
+                 RHRegistrationsListCustomize, methods=('GET', 'POST'))
+_bp.add_url_rule('/manage/registration/<int:reg_form_id>/registrations/static',
+                 'generate_static_url', RHRegistrationListStaticURL, methods=('POST',))
+_bp.add_url_rule('/manage/registration/<int:reg_form_id>/registrations/<int:registration_id>/', 'registration_details',
+                 RHRegistrationDetails)
+_bp.add_url_rule('/manage/registration/<int:reg_form_id>/registrations/<int:registration_id>/edit', 'edit_registration',
+                 RHRegistrationEdit)
```

Comment:
yeah you are right. I will update it though together with the next commit that will finish the task.